editors/vscode/package.json

```diff
@@ -135,7 +135,7 @@
 	"license": "MIT",
 	"scripts": {
 		"compile": "esbuild src/main.ts --bundle --outfile=out/main.js --external:vscode --format=cjs --platform=node --target=node14",
-		"watch": "pnpm run compile -- --sourcemap --watch",
```

Comment:
Missed this one in https://github.com/biomejs/biome/pull/219.

Let me know if you'd rather I submit this in a separate PR.